lib/ansible/utils/module_docs_fragments/vexata.py

```diff
@@ -0,0 +1,46 @@
+#
```

Comment:
Please rebase as this directory has moved to https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/doc_fragments/

Reply:
Will do.